settings.gradle.kts

```diff
@@ -0,0 +1,10 @@
+pluginManagement {
+    repositories {
+        gradlePluginPortal()
+    }
+}
+
+rootProject.name = "error-handling"
+include("core")
+include("avro")
+include("proto")
```

Comment:
In large-message and brute-force, we prefix the modules with the corresponding project. So in this case:
- `error-handling-core`
- `error-handling-avro`
- `error-handling-proto`

Otherwise, we would publish an artifact called `com.bakdata.kafka:avro`, which is too generic I think.